deps/rabbit/src/rabbit_deprecated_feature_extra.erl

```diff
@@ -0,0 +1,62 @@
+%% This Source Code Form is subject to the terms of the Mozilla Public
+%% License, v. 2.0. If a copy of the MPL was not distributed with this
+%% file, You can obtain one at https://mozilla.org/MPL/2.0/.
+%%
+%% @copyright 2018-2023 VMware, Inc. or its affiliates.
```

Comment:
The year should be 2023 only as the code didn't exist before.

Reply:
We have a mixture of copyrights along the code, most of them backdated like this. Also I just noticed a mix of VMware and Broadcom copyrights. @michaelklishin do we have a new convention to follow?

Reply:
The copyright holder is:
```
Broadcom. All Rights Reserved. The term “Broadcom” refers to Broadcom Inc. and/or its subsidiaries.  All rights reserved.
```